remoting/src/main/java/org/apache/rocketmq/remoting/exception/AuthenticationException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rocketmq.remoting.exception;
+
+/**
+ * Used when authorization failure , when Remoting component catches this exception, will ignore it.
```

Comment:
OK, I will modify it later. 

Besides, just a small questions about the commit process, should I squash the commits and recommit or just append a commit to let the admin to squash them?OK, I will modify it later. 

Besides, just a small questions about the commit process, should I squash the commits and recommit or just append a commit to let the admin to squash them?